uncompyle6/scanner.py

```diff
@@ -171,11 +171,11 @@ def bound_collection_from_tokens(
                 has_extended_arg=False,
             )
         )
-        if tokens[j] == "LOAD_CONST":
-            opname = "ADD_VALUE"
-        else:
-            opname = "ADD_VALUE_VAR"
         for j in range(collection_start, i):
+            if tokens[j] == "LOAD_CONST":
```

Comment:
I now realize that while this is necessary, it probably is not sufficient. There may be constants that are code blocks. 

However we can start at this and if you are up for it see if we can find those kinds of things that use LOAD_CONST with non-string items.